media-video/obs-studio/obs-studio-28.0.2.ebuild

```diff
@@ -0,0 +1,233 @@
+	x11-libs/libX11
+	x11-libs/libXcomposite
+	x11-libs/libXfixes
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	x11-libs/libxcb:=
+	alsa? ( media-libs/alsa-lib )
+	browser? (
+		app-accessibility/at-spi2-atk
+		app-accessibility/at-spi2-core:2
+		dev-libs/atk
+		dev-libs/expat
+		dev-libs/glib
+		dev-libs/nspr
+		dev-libs/nss
+		media-libs/alsa-lib
+		media-libs/fontconfig
+		media-libs/mesa[gbm(+)]
+		net-print/cups
+		x11-libs/cairo
+		x11-libs/libdrm
+		x11-libs/libXScrnSaver
+		x11-libs/libXcursor
+		x11-libs/libXdamage
+		x11-libs/libXext
+		x11-libs/libXi
+		x11-libs/libXrender
+		x11-libs/libXtst
+	)
+	fdk? ( media-libs/fdk-aac:= )
+	jack? ( virtual/jack )
+	lua? ( ${LUA_DEPS} )
+	pipewire? ( media-video/pipewire:= )
+	pulseaudio? ( media-sound/pulseaudio )
+	python? ( ${PYTHON_DEPS} )
+	speex? ( media-libs/speexdsp )
+	ssl? ( net-libs/mbedtls:= )
+	truetype? (
+		media-libs/fontconfig
+		media-libs/freetype
+	)
+	v4l? ( media-libs/libv4l )
+	vlc? ( media-video/vlc:= )
+	wayland? ( dev-libs/wayland )
+"
+RDEPEND="${DEPEND}"
+
+QA_PREBUILT="
+	usr/lib*/obs-plugins/chrome-sandbox
+	usr/lib*/obs-plugins/libcef.so
+	usr/lib*/obs-plugins/libEGL.so
+	usr/lib*/obs-plugins/libGLESv2.so
+	usr/lib*/obs-plugins/libvk_swiftshader.so
+	usr/lib*/obs-plugins/libvulkan.so.1
+	usr/lib*/obs-plugins/swiftshader/libEGL.so
+	usr/lib*/obs-plugins/swiftshader/libGLESv2.so
+"
+
+pkg_setup() {
+	use lua && lua-single_pkg_setup
+	use python && python-single-r1_pkg_setup
+}
+
+src_unpack() {
+	default
+
+	if [[ ${PV} == 9999 ]]; then
+		git-r3_src_unpack
+	else
+		if use browser; then
+			rm -d ${P}/plugins/obs-browser || die
+			mv obs-browser-${OBS_BROWSER_COMMIT} ${P}/plugins/obs-browser || die
+		fi
+
+		rm -d ${P}/plugins/obs-websocket || die
+		mv obs-websocket-${OBS_WEBSOCKET_COMMIT} ${P}/plugins/obs-websocket || die
+
+		rm -d ${P}/plugins/obs-websocket/deps/asio || die
+		mv asio-${ASIO_COMMIT} ${P}/plugins/obs-websocket/deps/asio || die
+
+		rm -d ${P}/plugins/obs-websocket/deps/json || die
+		mv json-${JSON_COMMIT} ${P}/plugins/obs-websocket/deps/json || die
+
+		rm -d ${P}/plugins/obs-websocket/deps/qr || die
+		mv QR-Code-generator-${QR_COMMIT} ${P}/plugins/obs-websocket/deps/qr || die
+
+		rm -d ${P}/plugins/obs-websocket/deps/websocketpp || die
+		mv websocketpp-${WEBSOCKETPP_COMMIT} ${P}/plugins/obs-websocket/deps/websocketpp || die
+	fi
+}
+
+src_configure() {
+	local libdir=$(get_libdir)
+	local mycmakeargs=(
+		$(usev browser -DCEF_ROOT_DIR=../${CEF_DIR})
+		-DBUILD_BROWSER=$(usex browser)
+		-DBUILD_VST=no
+		-DENABLE_WAYLAND=$(usex wayland)
+		-DENABLE_AJA=OFF
+		-DDISABLE_ALSA=$(usex !alsa)
+		-DDISABLE_DECKLINK=$(usex !decklink)
+		-DDISABLE_FREETYPE=$(usex !truetype)
+		-DDISABLE_JACK=$(usex !jack)
+		-DDISABLE_LIBFDK=$(usex !fdk)
+		-DENABLE_NEW_MPEGTS_OUTPUT=OFF
+		-DENABLE_PIPEWIRE=$(usex pipewire)
+		-DDISABLE_PULSEAUDIO=$(usex !pulseaudio)
+		-DDISABLE_SPEEXDSP=$(usex !speex)
+		-DDISABLE_V4L2=$(usex !v4l)
+		-DDISABLE_VLC=$(usex !vlc)
```

Comment:
A lot of these configuration flags have changed with 28 - I believe all of them are now ENABLE_*; see https://github.com/obsproject/obs-studio/commit/49e9d49943d501eb372dee1afa78919c3d8640d3

Reply:
Thanks, updated!